jinahub/encoders/audio/VGGISHAudioEncoder/vggish_audio_encoder.py

```diff
@@ -2,11 +2,16 @@
 __license__ = "Apache-2.0"
 
 import os
+from pathlib import Path
 from typing import Any, Optional, List, Iterable
+
+from jina import Executor, requests, DocumentArray
+from jina.logging.logger import JinaLogger
+import requests as requests_
```

Comment:
```suggestion
import requests as _requests
```

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/31/commits/d5752827dc4e83eb858cb90d75d24d2b01830e0e) for the change.